code/modules/mob/mob_helpers.dm

```diff
@@ -491,16 +491,18 @@ mob/proc/get_standard_bodytemperature()
 	. = list("[type]")
 
 
-/// Try to perform a unique action on the current active held item.
+/// Try to perform a unique action on the held items
 /mob/living/carbon/human/proc/do_unique_action()
 	SIGNAL_HANDLER
 	. = COMSIG_KB_ACTIVATED //The return value must be a flag compatible with the signals triggering this.
 	if(incapacitated() || lying_angle)
 		return
 
 	var/obj/item/active_item = get_active_held_item()
-	if(!istype(active_item))
-		return
+	if(istype(active_item))
+		active_item.unique_action(src)
+	var/obj/item/inactive_item =get_inactive_held_item()
+	if(istype(inactive_item))
+		inactive_item.unique_action(src)
```

Comment:
Missing return

Reply:
uh no that's intended, so it does exactly like the maro did

Reply:
What? So it will call unique action on BOTH of them at same time???

Reply:
like the maro did, yes

Reply:
i dont know if it did but this is way too inconvinient